Utilities/build-script-helper.py

```diff
@@ -51,8 +77,8 @@ def get_swiftpm_options(args):
       os.path.join(args.toolchain, 'lib', 'swift', 'Block'),
     ]
 
-  if 'ANDROID_DATA' in os.environ or (args.cross_compile_host and re.match(
-    'android-', args.cross_compile_host)):
+  if 'ANDROID_DATA' in os.environ or (args.cross_compile_hosts and re.match(
```

Comment:
Have you confirmed this line works? Matching a list doesn't seem to work for me.  Or maybe this is only working because it's actually a string right now (see my other comment on the argparse code).

```python
>>> re.match('as', 'asdf')
<re.Match object; span=(0, 2), match='as'>

>>> re.match('as', ['asdf'])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/AppleInternal/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/re.py", line 191, in match
    return _compile(pattern, flags).match(string)
TypeError: expected string or bytes-like object
```